src/jit/hwintrinsiclistxarch.h

```diff
@@ -11,58 +11,96 @@
 // clang-format off
 
 #if FEATURE_HW_INTRINSICS
-//                  Intrinsic ID                   Function name                ISA   
+//                 Intrinsic ID                         Function name                   ISA
 //  SSE Intrinsics          
-HARDWARE_INTRINSIC(SSE_IsSupported,             "get_IsSupported",              SSE)
-HARDWARE_INTRINSIC(SSE_Add,                     "Add",                          SSE)
+HARDWARE_INTRINSIC(SSE_IsSupported,                     "get_IsSupported",              SSE)
+HARDWARE_INTRINSIC(SSE_Add,                             "Add",                          SSE)
+HARDWARE_INTRINSIC(SSE_And,                             "And",                          SSE)
+HARDWARE_INTRINSIC(SSE_AndNot,                          "AndNot",                       SSE)
+HARDWARE_INTRINSIC(SSE_CompareEqual,                    "CompareEqual",                 SSE)
+HARDWARE_INTRINSIC(SSE_CompareGreaterThan,              "CompareGreaterThan",           SSE)
+HARDWARE_INTRINSIC(SSE_CompareGreaterThanOrEqual,       "CompareGreaterThanOrEqual",    SSE)
+HARDWARE_INTRINSIC(SSE_CompareLessThan,                 "CompareLessThan",              SSE)
+HARDWARE_INTRINSIC(SSE_CompareLessThanOrEqual,          "CompareLessThanOrEqual",       SSE)
+HARDWARE_INTRINSIC(SSE_CompareNotEqual,                 "CompareNotEqual",              SSE)
+HARDWARE_INTRINSIC(SSE_CompareNotGreaterThan,           "CompareNotGreaterThan",        SSE)
+HARDWARE_INTRINSIC(SSE_CompareNotGreaterThanOrEqual,    "CompareNotGreaterThanOrEqual", SSE)
+HARDWARE_INTRINSIC(SSE_CompareNotLessThan,              "CompareNotLessThan",           SSE)
+HARDWARE_INTRINSIC(SSE_CompareNotLessThanOrEqual,       "CompareNotLessThanOrEqual",    SSE)
+HARDWARE_INTRINSIC(SSE_CompareOrdered,                  "CompareOrdered",               SSE)
+HARDWARE_INTRINSIC(SSE_CompareUnordered,                "CompareUnordered",             SSE)
+HARDWARE_INTRINSIC(SSE_Divide,                          "Divide",                       SSE)
+HARDWARE_INTRINSIC(SSE_Load,                            "Load",                         SSE)
+HARDWARE_INTRINSIC(SSE_LoadAligned,                     "LoadAligned",                  SSE)
+HARDWARE_INTRINSIC(SSE_Max,                             "Max",                          SSE)
+HARDWARE_INTRINSIC(SSE_Min,                             "Min",                          SSE)
+HARDWARE_INTRINSIC(SSE_MoveHighToLow,                   "MoveHighToLow",                SSE)
+HARDWARE_INTRINSIC(SSE_MoveLowToHigh,                   "MoveLowToHigh",                SSE)
+HARDWARE_INTRINSIC(SSE_Multiply,                        "Multiply",                     SSE)
+HARDWARE_INTRINSIC(SSE_Or,                              "Or",                           SSE)
+HARDWARE_INTRINSIC(SSE_Reciprocal,                      "Reciprocal",                   SSE)
+HARDWARE_INTRINSIC(SSE_ReciprocalSqrt,                  "ReciprocalSquareRoot",         SSE)
```

Comment:
This name is getting fixed in https://github.com/dotnet/coreclr/pull/15471, will update after it is merged.